ViewModeSwitch/index.html

```diff
@@ -9,8 +9,7 @@
 		<meta name="keywords" content="view mode, switch, css, style, grid, list, template" />
 		<meta name="author" content="Codrops" />
 		<link rel="shortcut icon" href="../favicon.ico">
-		<link rel="stylesheet" type="text/css" href="css/default.css" />
-		<link rel="stylesheet" type="text/css" href="css/component.css" />
+		<link rel="stylesheet" type="text/css" href="css/css/styles.css" />
```

Comment:
Good catch @kellymurray.  That was some stray copy pasta from trying to delete the component.css and plug in the styles.css, but I must have copied the "css/" of the styles.css name.